### ISSUE TITLE
[BUG] gtk qt Ubuntu rendering and input method bugs

### ISSUE THREAD

**Issue by InfernalAzazel** [r0x0r/pywebview#1052]
### Specification
- pywebview version: 4.0.1
- operating system:
- web renderer: Ubuntu 20.04 tls

### Description  
This is the normal state of the browser:

![图片](https://user-images.githubusercontent.com/40333954/219627786-48a9dfdd-f3d7-4ba2-b58a-4237dc52ebd5.png)

I used gtk ui:

bug: The element on the left is not out, there are three sessions
  
![图片](https://user-images.githubusercontent.com/40333954/219627395-084d6586-01ad-4a0f-83b8-e48f133c321c.png)

I used qt ui: 

bug: 

1.Unable to pop up the keyboard input method

https://user-images.githubusercontent.com/40333954/219632266-7a0bfd38-ea26-478c-9ee1-be63b9aa82da.mp4

2.webview.start debug =True Program crash

```
Traceback (most recent call last):
  File "/home/V01/extittivns03/.pyenv/versions/3.10.8/lib/python3.10/site-packages/webview/platforms/qt.py", line 144, in show_inspector
    BrowserView.instances[uid].raise_()
KeyError: 'master-inspector'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/V01/extittivns03/.pyenv/versions/3.10.8/lib/python3.10/site-packages/webview/platforms/qt.py", line 150, in show_inspector
    window = Window('web_inspector', title, url, '', 700, 500, None, None, True, False,
TypeError: Window.__init__() missing 1 required positional argument: 'localization'
http://localhost:8228

```

### Practicalities

- NO I am willing to work on this issue myself.

- NO I am prepared to support this issue financially.

**Comment by r0x0r** [comment]
GTK:
My guess it is because of old renderer used by GTK. No workaround for this other than use another renderer

QT
1. I don't follow what the problem is 
2. Fixed in 4.0.2

**Comment by InfernalAzazel** [comment]
QT
1. Because it will take up all the basic functions of other shortcut keys

**Comment by InfernalAzazel** [comment]
> GTK： 我猜这是因为 GTK 使用了旧的渲染器。 除了使用另一个渲染器之外没有解决方法
> 
> QT
> 
>     1. I don't follow what the problem is
> 
>     2. Fixed in 4.0.2

man, qt 2.The problem is indeed fixed

**Comment by InfernalAzazel** [comment]
Unable to pop up the keyboard input method

![20230223-101139](https://user-images.githubusercontent.com/40333954/220807923-59a0ed71-0b8c-4098-a24e-f50c2973f944.gif)
